src/ui/BookmarksComboBoxWidget.cpp

```diff
@@ -97,6 +97,7 @@ void BookmarksComboBoxWidget::setCurrentFolder(BookmarksModel::Bookmark *folder)
 	if (folder)
 	{
 		setCurrentIndex(folder->index());
+		getView()->expand(folder->index());
```

Comment:
Just a minor detail, usually we put empty newlines between groups of calls dealing with different objects.
Also, I'm not 100% sure,but  I kind of expected that it would be needed to call it on parent index instead, does it still work properly?

Reply:
@Emdek still works properly calling it on child.

Reply:
@fnkkio, OK.